testing/regression/gh_902__iterator_category_to_system_returns_wrong_tag.cu

```diff
@@ -0,0 +1,22 @@
+#include <thrust/iterator/detail/iterator_category_with_system_and_traversal.h>
+#include <thrust/iterator/iterator_categories.h>
+#include <thrust/iterator/iterator_traits.h>
+
+template <class IteratorCategory>
+using _category_to_system_t =
+    typename thrust::detail::iterator_category_to_system<IteratorCategory>::type;
+
+static_assert(std::is_same<thrust::device_system_tag, _category_to_system_t<thrust::output_device_iterator_tag>>::value);
+static_assert(std::is_same<thrust::device_system_tag, _category_to_system_t<thrust::input_device_iterator_tag>>::value);
+// static_assert(std::is_same<thrust::device_system_tag, _category_to_system_t<thrust::forward_device_iterator_tag>>::value);
```

Comment:
Are the commented out examples still broken? Let's either enable them or add a comment explaining why they're disabled.